src/Server/Interfaces/ITelnetCodeHandler.cs

```diff
@@ -9,13 +9,13 @@
 
 namespace WheelMUD.Interfaces
 {
-    using System.Collections.Generic;
-
     /// <summary>An interface defining a TelnetCodeHandler.</summary>
     public interface ITelnetCodeHandler
     {
-        /// <summary>Gets a list of the telnet options recognised by the server.</summary>
-        List<ITelnetOption> TelnetOptions { get; }
+        /// <summary>Find the TelnetOption class instance of the given type.</summary>
```

Comment:
Because I had inserted a new property, then removed that property. Git just kept a shadow of the change, even though I reverted it back to the way it originally was. No actual change here.